platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatDarkLaf.properties

```diff
@@ -22,6 +22,10 @@ nb.preferred.color.profile=FlatLaf Dark
 nb.errorForeground=#DB5860
 nb.warningForeground=@foreground
 
+# Keep the tree background at @background rather than the very slightly lighter
+# @componentBackground, to make the Projects/Files/Services and Navigator backgrounds connect
+# seamlessly with the background of their tab on top.
+Tree.background = @background
```

Comment:
It would affect JTree, OutlineView etc. in dialog boxes as well (though it's a rather small color difference). If you prefer, I could add a separate property instead that is explicitly queried by the Projects, Files, Services, and Navigation panes.

What is unified background support? Is that the same as uniform title bar? (The uniform title bar is shown in the screenshots.)

Reply:
I just did a search for uses of `Tree.background` across the codebase.  eg. there's https://github.com/apache/netbeans/blob/master/platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLFCustoms.java#L159 It's probably fine, but might be worth checking.

Reply:
I think it's the general Swing JTree UI property, and the NetBeans TreeView (used for the Projects pane and others) uses a JTree behind the scenes.

Reply:
Yes, it's the general Swing tree background. Its value is used in a few other places for other things. Probably not adversely affected. Just wondered if you'd checked?

Reply:
Thanks. I realise they're very similar colours, but still noticeably different next to each other (otherwise you wouldn't need to make the change).